File: src/com/LilG/Com/RPSGame.java
package com.LilG.Com;

import org.jetbrains.annotations.NotNull;

/**
 * Created by ggonz on 12/13/2015.
 */
@SuppressWarnings("SameParameterValue")
class RPSGame {
	private String player1;
	private String player2;

	private int choice1 = 0;
	private int choice2 = 0;

	public RPSGame(String player1, String player2){
		this.player1 = player1;
		this.player2 = player2;
	}

	public boolean isInGame(@NotNull String playerToCheck) {
		return playerToCheck.equalsIgnoreCase(player1) || playerToCheck.equalsIgnoreCase(player2);
	}

	public boolean isFirstPlayer(@NotNull String playerToCheck) {
		return playerToCheck.equalsIgnoreCase(player1);
	}

	public void setP1Choice(int choice){
		this.choice1 = choice;
	}

	public void setP2Choice(int choice){
		this.choice2 = choice;
	}

	public boolean isGameSet(){
		return choice1 != 0 && choice2 != 0;
	}

}

File: FozruciCS/Misc/RPSGame.cs
using ikvm.extensions;

namespace FozruciCS.Misc{
	public struct RpsGame{
		private readonly string _player1;
		private readonly string _player2;

		private int _choice1;
		private int _choice2;

		public RpsGame(string player1, string player2) {
			_player1 = player1;
			_player2 = player2;
			_choice1 = 0;
			_choice2 = 0;
		}

		public bool isInGame(string playerToCheck) {
			return playerToCheck.equalsIgnoreCase(_player1) || playerToCheck.equalsIgnoreCase(_player2);
		}

		public bool isFirstPlayer(string playerToCheck) {
			return playerToCheck.equalsIgnoreCase(_player1);
		}

		public void setP1Choice(int choice) {
			_choice1 = choice;
		}

		public void setP2Choice(int choice) {
			_choice2 = choice;
		}

		public bool isGameSet() {
			return (_choice1 != 0) && (_choice2 != 0);
		}
	}
}